src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -86,12 +88,28 @@ public function getSchema(): Schema
             $graphqlConfiguration = $resourceMetadata->getGraphql() ?? [];
             foreach ($graphqlConfiguration as $operationName => $value) {
                 if ('query' === $operationName) {
-                    $queryFields += $this->getQueryFields($resourceClass, $resourceMetadata);
+                    $queryFields += $this->getQueryFields($resourceClass, $resourceMetadata, $operationName, ['args' => ['id' => ['type' => GraphQLType::id()]]], true);
```

Comment:
Why `true` for the `$collection` parameter?

Reply:
Maybe an empty array will be better IMO.

Reply:
Because that is the default behaviour. `query` generates both an item and a collection query (using the default input object).

Reply:
Your choice. Would just need to modify the condition inside `getQueryFields` to explicitly check for `!== false` instead of "truthy" ;)

Reply:
I think it would be better since it would also remove the `is_array` checks :wink:

Reply:
Makes sense 👍